.github/workflows/sipnet.yml

```
@@ -49,16 +49,15 @@ jobs:
       run: ./scripts/add.models.sh
 
     # install sipnet
-    - name: Check out SIPNET
-      uses: actions/checkout@v4
-      with:
-        repository: PecanProject/sipnet
-        path: sipnet
-        set-safe-directory: false
-    - name: install sipnet
+    - name: Download SIPNET for Linux
+      run: |
+        curl -L -o sipnet-linux "https://github.com/PecanProject/sipnet/releases/download/v1.3.0/sipnet-linux-v1.3.0"
+        chmod +x sipnet-linux
+
+    - name: Download climate file from GitHub
       run: |
-        cd ${GITHUB_WORKSPACE}/sipnet
-        make sipnet
+        curl -L -o niwot.clim "https://raw.githubusercontent.com/PecanProject/sipnet/refs/heads/master/Sites/Niwot/niwot.clim"
```

Apologies that we moved the ground under you here -- @Alomir and @dlebauer just did a big (and overdue, thank you!) reorganization of the Sipnet repository, and this file has moved. I think for this PR it's enough to change the path:

```suggestion
        curl -L -o niwot.clim "https://raw.githubusercontent.com/PecanProject/sipnet/refs/heads/master/tests/smoke/niwot.clim"
```


Long term there's no guarantee the Sipnet repo will keep providing this file, so we may want to consider providing it somewhere under PEcAn's control, but this works for now.